OSDOCS-16409: Cluster Admin CQAs (#116629)
openshift/openshift-docs: modules/dedicated-aws-vpn-creating.adoc

[id="dedicated-aws-vpn-creating"]
= Creating a VPN connection

[role="_abstract"]
You can configure an Amazon Web Services (AWS) {product-title} cluster to use a customer's on-site hardware VPN device using the following procedures.

.Prerequisites

* Hardware VPN gateway device model and software version, for example Cisco ASA running version 8.3. See the Amazon VPC link:https://docs.aws.amazon.com/vpc/latest/adminguide/Introduction.html#DevicesTested[Network Administrator Guide] to confirm whether your gateway device is supported by AWS.
* Public, static IP address for the VPN gateway device.

> * BGP or static routing: if BGP, the ASN is required. If static routing, you must configure at least one static route.

Reviewer: If we aren't removing this module, same as above:

```suggestion
* Border Gateway Protocol (BGP) or static routing: if using (or if you choose) BGP, the Autonomous System Number (ASN) is required. If static routing, you must configure at least one static route.
```

.. Select the newly created gateway.
.. Select *Actions* from the list and click *Attach to VPC*.
.. Select the newly created gateway under Available VPC’s and click *Attach to VPC* to attach it to the cluster VPC you identified earlier.
. Establish the VPN connection. From the VPC dashboard, under Virtual private network (VPN) click on *Site-to-Site VPN connections*.
. Click *Create VPN connection*.
.. Give it a meaningful name tag.
.. Select the Virtual private gateway created previously.
.. For Customer gateway, select *Existing*.
.. Select the Customer gateway id by name.
.. If the VPN will use BGP, select *Dynamic*, otherwise select *Static* and enter the Static IP CIDRs. If there are multiple CIDRs, add each CIDR as *Another Rule*.
.. Click *Create VPN connection*.
.. Under *State* wait for the VPN status to change from *Pending* to *Available*, approximately 5 to 10 minutes.
. Select the VPN you just created and click *Download configuration*.
.. From the list, select the vendor, platform, and version of the customer gateway device, then click *Download*.
.. The *Generic* vendor configuration is also available for retrieving information in a plain text format.
+
[NOTE]
====

> After the VPN connection has been established, be sure to set up Route Propagation or the VPN may not function as expected. Note the VPC subnet information, which you must add to your configuration as the remote network. 

Reviewer: ```suggestion
After the VPN connection has been established, set up Route Propagation or the VPN might not function as expected. Note the VPC subnet information, which you must add to your configuration as the remote network. 
```
Forgot the reason why, but the rule says "Do not use may as a synonym for might." (https://redhat.atlassian.net/wiki/spaces/CCSENABLE/pages/129467858/Avoid+these+words)
